src/hooks/GetFlights.js

```diff
@@ -0,0 +1,25 @@
+import firestore from '@react-native-firebase/firestore';
+import auth from '@react-native-firebase/auth';
+
+const GetFlights = async () => {
+  const currentUser = auth().currentUser;
+  try{
+    const flightsCollection = await firestore()
+    .collection('flights')
+    .where('userId', '==', currentUser.uid)
+    .get();
+    const flights = [];
+    flightsCollection.forEach(documentSnapshot => {
+      const flight = documentSnapshot.data();
+      flights.push(flight);
+    });
+    console.log(flights);
+    return flights;
+  }
+  catch(error){
+    console.log(error);
```

Comment:
Don't leave debug code on GitHub, like console.log(), you can use Alert.alert() instead